tests/test_convert.py

```diff
@@ -11,111 +9,88 @@
 import audb
 
 
-os.environ['AUDB_CACHE_ROOT'] = pytest.CACHE_ROOT
-os.environ['AUDB_SHARED_CACHE_ROOT'] = pytest.SHARED_CACHE_ROOT
-
-
-@pytest.fixture(
-    scope='session',
-    autouse=True,
-)
-def fixture_set_repositories():
-    audb.config.REPOSITORIES = pytest.REPOSITORIES
-
-
-DB_NAME = f'test_convert-{pytest.ID}'
-DB_ROOT = os.path.join(pytest.ROOT, 'db')
-
-DB_FILES = {
-    'audio/file1.wav': {
-        'bit_depth': 16,
-        'channels': 1,
-        'format': 'wav',
-        'sampling_rate': 8000,
-    },
-    'audio/file2.wav': {
-        'bit_depth': 24,
-        'channels': 2,
-        'format': 'wav',
-        'sampling_rate': 16000,
-    },
-    'audio/file3.flac': {
-        'bit_depth': 8,
-        'channels': 3,
-        'format': 'flac',
-        'sampling_rate': 44100,
-    },
-}
-
-
-def clear_root(root: str):
-    root = audeer.path(root)
-    if os.path.exists(root):
-        shutil.rmtree(root)
+DB_NAME = 'test_convert'
```

Comment:
Now that we use temp directories everywhere, we could probably use the same name everywhere if we want. But I'm also fine with having a different name in each test file.

Reply:
Yes, I think it's fine like it is.